.github/workflows/androidTestSuite.yml

```diff
@@ -0,0 +1,126 @@
+name: Android Test Suite
```

Comment:
better to split the jobs for unit tests and Instrument tests (including Facebook screenshot tests).